lib/nostrum/api.ex

```diff
@@ -749,10 +749,17 @@ defmodule Nostrum.Api do
     do: send_chunked_delete(messages, channel_id)
 
   def bulk_delete_messages(channel_id, messages, true) do
+    alias Nostrum.Struct.Snowflake
+
+    snowflake_two_weeks_ago =
+      DateTime.utc_now()
+      |> DateTime.to_unix()
+      |> Kernel.-(60 * 60 * 24 * 14)
```

Comment:
Could we add a comment here that this is supposed to represent 2 weeks?

Reply:
Added in 861ab4afc1679ad212cb849a017c8995886a3554.